utils/rename_tensors.py

```diff
@@ -0,0 +1,266 @@
+import numpy as np
+
+import equistore
+from equistore import Labels, TensorBlock, TensorMap
+
+
+def get_tensorblock_name_structure(tensorblock: equistore.block.TensorBlock) -> dict:
+    """
+    Takes a TensorBlock object and returns a dict showing the
+    structure of the Label names. An example is shown below for
+    a descriptor where 'cell' and 'positions' gradients have been
+    calculated:
+
+    {
+    "samples": ("structure", "center"),
+    "components": [("spherical_harmonics_m",)],
+    "properties": ("n",),
+    "gradients": {
+        "cell": {
+            "samples": ("sample",),
+            "components": [
+                ("direction_1",),
+                ("direction_2",),
+                ("spherical_harmonics_m",),
+            ],
+            "properties": ("n",),
+        },
+        "positions": {
+            "samples": ("sample", "structure", "atom"),
+            "components": [("direction",), ("spherical_harmonics_m",)],
+            "properties": ("n",),
+        },
+    },
+    }
+
+    Parameters
+    ----------
+    tensorblock : equistore.block.TensorBlock
+        The input TensorBlock object whose Label name structure will be returned
+
+    Returns
+    -------
+    dict
+        The name structure of the input TensorBlock.
+    """
+    names = {
+        "samples": tensorblock.samples.names,
+        "components": [c.names for c in tensorblock.components],
+        "properties": tensorblock.properties.names,
+    }
+    if type(tensorblock) == equistore.block.TensorBlock:
```

Comment:
this is a bit weird. why do you need this check? 

Also, this pattern is usually written `isinstance(tensorblock, equistore.block.TensorBlock)` (to support inheritance).

Reply:
Change to ``isinstance``.